themes/default/content/docs/concepts/vs/terraform/_index.md

```diff
@@ -80,7 +80,7 @@ Terraform provides open-source [infrastructure as code](/what-is/what-is-infrast
 
 ## Pulumi vs. Terraform: Similarities {#similarities}
 
-Terraform and Pulumi have some similarities in that you can create, deploy and manage infrastructure as code on any cloud. Like Terraform, Pulumi is [open source on GitHub](https://github.com/pulumi/pulumi) and is [free to use](/docs/get-started/). Both Terraform and Pulumi offer a desired state infrastructure as code model where the code represents the desired infrastructure state and the deployment engine compares this desired state with the stack’s current state and determines what resources need to be created, updated or deleted.  Both Terraform and Pulumi support many cloud providers, including [AWS](/aws/), [Azure](/azure/), and [Google Cloud](/gcp/), plus other services like CloudFlare, Digital Ocean, and more. Thanks to integration with Terraform providers, Pulumi is also able to support a superset of the providers that Terraform currently offers.
+Terraform and Pulumi have some similarities in that you can create, deploy and manage infrastructure as code on any cloud. Terraform has adopted a Business Source License model, while Pulumi is [open source on GitHub](https://github.com/pulumi/pulumi) and is [free to use](/docs/get-started/). Both Terraform and Pulumi offer a desired state infrastructure as code model where the code represents the desired infrastructure state and the deployment engine compares this desired state with the stack’s current state and determines what resources need to be created, updated or deleted.  Both Terraform and Pulumi support many cloud providers, including [AWS](/aws/), [Azure](/azure/), and [Google Cloud](/gcp/), plus other services like CloudFlare, Digital Ocean, and more. Thanks to integration with Terraform providers, Pulumi is also able to support a superset of the providers that Terraform currently offers.
```

Comment:
I'm going to suggest that licensing discussion should be #1 on the list of "key differences". This is now a key differentiator that we will be turning up the volume on.

Reply:
Great call out!